modules/custom/az_seo/az_seo.info.yml

```diff
@@ -9,3 +9,4 @@ dependencies:
   - metatag
   - metatag_open_graph
   - metatag_twitter_cards
+  - pathauto
```

Comment:
the `pathauto` dependency should be here in `az_seo` instead of `az_quickstart.info.yml`